Lombiq.HelpfulLibraries.LinqToDb/Docs/LinqToDb.md

```diff
@@ -0,0 +1,45 @@
+# Lombiq Linq2DB for Orchard Core

```

Comment:
Ebben a fájlban kéne némi formázási clean-up, lásd: https://lombiq.atlassian.net/wiki/spaces/DEV/pages/12156958/Adding+documentation+pages+to+our+projects